src/site/content/en/blog/better-youtube-web-part1/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Building a Better Web - Part 1: A faster YouTube on web'
+subhead: >
+  A case study of changes the YouTube Web team made to improve performance, increase their Core Web Vitals pass rates and lift key business metrics.
+description: >
+  This is Part 1 of our new series on Building a Better YouTube Web called "Building a faster YouTube on web"
+layout: post
+date: 2022-10-19
+authors:
+  - addyosmani
+  - sriramkrishnan
+hero: 'image/1L2RBhCLSnXjCnSlevaDjy3vba73/CPrVgVl7t1MuwWpXzvCE.jpg'
+alt: >
+  A hero image of the YouTube mobile web loading experience.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - case-study
+  - performance
+  - web-vitals
+---
+
+The Chrome team often talks about "building a *better* web", but what does that
+mean? Web experiences should be [fast](/why-speed-matters/),
+[accessible](/accessibility/), and leverage [device
+capabilities](/reliable/) in the moment when users need it most.
+Dogfooding is part of Google's culture, so the Chrome team has partnered with
```

Comment:
(Optional) Maybe link to this for the word "Dogfooding":
https://en.wikipedia.org/wiki/Eating_your_own_dog_food

Reply:
yes please.

Reply:
SGTM